RomM.cs

```diff
@@ -235,74 +243,89 @@ public override IEnumerable<GameMetadata> GetGames(LibraryGetGamesArgs args)
+                    Stream body = response.Content.ReadAsStreamAsync().GetAwaiter().GetResult();
+                    List<RomMRom> roms;
+                    using (StreamReader reader = new StreamReader(body))
+                    using (JsonTextReader jsonReader = new JsonTextReader(reader))
+                    {
+                        roms = JsonSerializerSingleton.Instance.Deserialize<List<RomMRom>>(jsonReader);
+                    }
+
+                    Logger.Debug($"Finished parsing response for {apiPlatform.Name}.");
+
+                    var installDir = PlayniteApi.Paths.IsPortable
+                        ? mapping.DestinationPathResolved.Replace(PlayniteApi.Paths.ApplicationPath, ExpandableVariables.PlayniteDirectory)
+                        : mapping.DestinationPathResolved;
 
                     // Return a GameMetadata for each item in the response
-                    foreach (var item in items)
+                    foreach (var item in roms)
                     {
-                        var gameName = item["name"].ToString();
-                        var fileName = item["file_name"].ToString();
-                        var urlCover = item["url_cover"]?.ToString() ?? null;
+                        var gameName = item.Name;
+                        var fileName = item.FileName;
+                        var urlCover = item.UrlCover;
                         var pathToGame = Path.Combine(installDir, fileName);
 
-                        var info = new RomMGameInfo()
+                        var info = new RomMGameInfo
                         {
                             MappingId = mapping.MappingId,
                             FileName = fileName,
-                            DownloadUrl = $"{Settings.RomMHost}/api/roms/{item["id"].ToObject<int>()}/content/{fileName}",
-                            IsMulti = item["multi"].ToObject<bool>()
+                            DownloadUrl = $"{Settings.RomMHost}/api/roms/{item.Id}/content/{fileName}",
+                            IsMulti = item.Multi
                         };
                         var gameId = info.AsGameId();
                         responseGameIDs.Add(gameId);
 
                         // Check if the game is already installed
-                        if (Playnite.Database.Games.Where(g => g.GameId == gameId).Any())
+                        if (Playnite.Database.Games.Any(g => g.GameId == gameId))
                         {
                             continue;
                         }
 
+                        var gameNameExtended = $"{gameName}{(item.Regions.Count > 0 ? $" ({string.Join(", ", item.Regions)})" : "")}{(!string.IsNullOrEmpty(item.Revision) ? $" (Rev {item.Revision})" : "")}{(item.Tags.Count > 0 ? $" ({string.Join(", ", item.Tags)})" : "")}";
```

Comment:
i think `gameNameWithTags` would be clearer

Reply:
changed in 26b7f9d7311af249172a59a9b72f62dfff558c60 😄